docs/src/content/docs/fr/guides/css-and-tailwind.mdx

```diff
@@ -252,9 +252,11 @@ Ces variables sont utilisées dans toute l'interface utilisateur avec une gamme
 
 ### Éditeur de thème de couleur
 
-Utiliser les contrôles ci-dessous pour modifier les palettes de couleurs d'accentuation et de gris de Starlight.
+Utilisez les contrôles ci-dessous pour modifier les palettes de couleurs d'accentuation et de gris de Starlight.
 Les zones d'aperçu sombre et clair afficheront les couleurs résultantes, et la page entière sera également mise à jour pour prévisualiser vos modifications.
 
+Utilisez l'option Niveau de contraste pour spécifier lequel des [standards de contraste de couleur](https://developer.mozilla.org/fr/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) des règles pour l’accessibilité des contenus Web à respecter.
```

Comment:
Source for "Web Content Accessibility Guideline": https://www.w3.org/WAI/standards-guidelines/wcag/fr

Reply:
And I thought “WCAG” was hard to pronounce 😁